lib/Agrammon/Model.pm6

```diff
@@ -306,7 +306,8 @@ class Agrammon::Model {
     method dump {
         my Str $output;
         for @!evaluation-order.reverse {
-            $output ~= $_.taxonomy ~ "\n";
+            my $level = ($_.taxonomy ~~ m:global/ '::' /).elems;
+            $output ~= $_.taxonomy.indent(4 * $level) ~ "\n";
```

Comment:
`$_.taxonomy` can be spelled `.taxonomy` (in line with the general principle that much of the time, you don't have to explicitly mention `$_`).

Reply:
I stole that from a few lines above without really thinking if the `$_`  is really needed here.